extensions-contrib/druid-iceberg-extensions/pom.xml

```diff
@@ -35,6 +35,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <properties>
+    <iceberg.core.version>1.4.0</iceberg.core.version>
```

Comment:
```suggestion
    <iceberg.core.version>1.4.1</iceberg.core.version>
```

Reply:
I'm currently testing 1.4.1 and will be upgrading to it as part of another PR. Thanks.

Reply:
@a2l007 No problem, thanks a lot